node/src/services/handler.rs

```diff
@@ -291,5 +292,22 @@ where
             .clone()
             .subscribe_to(&self.task_spawner, &self.blockchain);
         sp_stop_storing_insolvent_user_event_bus_listener.start();
+
+        // Delete file and update the root.
```

Comment:
Probably worth a bit more description here. Like the fact that this is the task that removes a file as a consequence of this BSP requesting to stop storing it.